File: experiments/polyglot/services/cpp/service_test.cpp
#include "identity.hpp"
#include "azimuth.hpp"

#include <cassert>
#include <string>

AZIMUTH_COVERS("polyglot/identity", "cpp-identifies", "unit", "example", "direct")
int main() {
    assert(std::string(identity()) == "cpp");
}

File: experiments/polyglot/services/javascript/service.test.js
const assert = require('node:assert/strict');
const { test } = require('node:test');
const { covers } = require('../../../../packages/typescript/dist/index.js');
const { identity } = require('./service.js');

test('identity is javascript', () => {
  covers('polyglot/identity', 'javascript-identifies', 'unit', 'example', 'direct');
  assert.equal(identity(), 'javascript');
});
